stash/package.json

```diff
@@ -31,14 +31,12 @@
   },
   "dependencies": {
     "@google-cloud/logging-winston": "^5.3.0",
-    "@mangata-finance/sdk": "2.2.1-eth-rollup-develop.27",
-    "@mangata-finance/types": "2.2.3-eth-rollup-develop.34",
-    "@polkadot/api": "10.12.2",
+    "@polkadot/api": "^11.1.1",
```

Comment:
@polkadot/api version change is because of the change of polkadot/api change in gasp-dev-kit and all the other version changes and resolutions in this package.json are related to that.